contract_subscriber/index.js

```diff
@@ -22,15 +22,6 @@ async function handler() {
 				await writeAsync('currentBlock', eventBlock);
 			}
 
-			// ETHProfiles latest events
-			fromBlock = await getAsync('ethProfilesCurrentBlock') || 0;
-			events = await ETHProfiles.getPastEvents({fromBlock, toBlock: 'latest'});
```

Comment:
As we're not using the `ETHProfiles` anymore, we can remove the import on line https://github.com/Bounties-Network/BountiesAPI/pull/123/files#diff-6cd342a29084c0f8530057d1ad74994fR5

Reply:
good call, thanks @codeluggage!